docs/using-connector.md

```diff
@@ -533,6 +534,160 @@ conf.set("spark.riakts.write.bulk-size", "500")
+            obj
+        }.saveToRiak(namespace)
+      }
+```
+ And finally, we are starting the stream:
+ 
+ ```scala
+ streamCtx.start()
+ streamCtx.awaitTermination()
+ ```
+
+ Now that we understand the code, we can run the `StreamingKVExample.scala` example with:
+ 
+ ```
+ /path/to/spark-riak-connector-examples/bin/run-example streaming.StreamingKVExample
+ ```
+ 
+ This wil start a stream from the Kafka topic `streaming` into the KV bucket `test-data` that we just created. This stream will run until terminated. Whenever a message is produced for Kafka topic `streaming`, the spark streaming context that the example creates will automatically stream the message from the topic into the KV bukcet. To see this in action, we need to send a message to the kafka topic `streaming` with the kafka console producer script:
+ 
+ ```
+ /path/to/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streaming
+ ```
+ 
+ This will read a message from standard in and pass it to the topic. As an example put the following into standard in:
+ 
+ ```
+ {"time": "2016-01-01 08:30:00.000", "weather": "sunny", "temperature": 25.0, "humidity": 67.0, "pressure": 30.20, "family": "f"}
+ ```
+ 
+You should now be able to see this data entry in the KV bucket `test-data`.
+
+###Spark Streaming TS Table Example
+
+Having seen how spark streaming works with KV buckets, lets now look at the TS table example [here](../examples/src/main/scala/com/basho/riak/spark/examples/streaming/StreamingTSExample.scala). 
+
+The code is somewhat similar to the KV bucket example, but with crucial differences. Let have a look:
+
+```scala
+ val schema = StructType(List(
+      StructField(name = "weather", dataType = StringType),
+      StructField(name = "family", dataType = StringType),
+      StructField(name = "time", dataType = TimestampType),
+      StructField(name = "temperature", dataType = DoubleType),
+      StructField(name = "humidity", dataType = DoubleType),
+      StructField(name = "pressure", dataType = DoubleType)))
+
+    val sparkConf = new SparkConf(true)
+      .setAppName("Simple Spark Streaming to Riak TS Demo")
+
+    setSparkOpt(sparkConf, "spark.master", "local")
+    setSparkOpt(sparkConf, "spark.riak.connection.host", "127.0.0.1:8087")
+    setSparkOpt(sparkConf, "kafka.broker", "127.0.0.1:9092")
+
+    val sc = new SparkContext(sparkConf)
+    val streamCtx = new StreamingContext(sc, Durations.seconds(15))
+    val sqlContext = new org.apache.spark.sql.SQLContext(sc)
+    import sqlContext.implicits._
+
+    val kafkaProps = Map[String, String](
+      "metadata.broker.list" -> sparkConf.get("kafka.broker"),
+      "client.id" -> UUID.randomUUID().toString
+    )
+```
+
+This first section of code just sets up the table schema, a spark streaming context, a spark sql context, and Kafka properties. Note that we need to set up a table in TS that reflect the schema in the previous section of code. We can create this table in TS with:
```

Comment:
capitalize both words in "spark streaming"
change "spark sql" to "Spark SQL"
change "TS" to "Riak TS"